### PR TITLE
satellitedb: bucket search fixed

### PR DESCRIPTION
**Pull request by VitaliiShpital**
What: 
bucket search fixed. Search request on existing bucket now returns expected result

Why:
Search request on existing bucket returned nothing

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/satellitedb/projectaccounting.go [hunk 1]
```diff
@@ -329,7 +329,7 @@ func (db *ProjectAccounting) GetBucketTotals(ctx context.Context, projectID uuid
 	countQuery := db.db.Rebind(`SELECT COUNT(DISTINCT bucket_name)
 		FROM bucket_bandwidth_rollups
 		WHERE project_id = ? AND interval_start >= ? AND interval_start <= ?
-		AND CAST(bucket_name as TEXT) LIKE ?`)
```
Comment (crawter) [inline satellite/satellitedb/projectaccounting.go]: bucket_name is a bytea. I think you should convert/encode it to text before comparing
try to use ```encode(data bytea, format text)```

Comment (rikysya) [reply satellite/satellitedb/projectaccounting.go]: much better solution is to encode `search` string into bytes

File: satellite/satellitedb/projectaccounting.go [hunk 2]
```diff
@@ -426,15 +426,15 @@ func (db *ProjectAccounting) GetBucketTotals(ctx context.Context, projectID uuid
 	bucketsQuery := db.db.Rebind(`SELECT DISTINCT bucket_name
 		FROM bucket_bandwidth_rollups
 		WHERE project_id = ? AND interval_start >= ? AND interval_start <= ?
-		AND CAST(bucket_name as TEXT) LIKE ?
+		AND bucket_name LIKE ?
```
Comment (crawter) [inline satellite/satellitedb/projectaccounting.go]: is this query safe from sql injection?

Comment (crawter) [reply satellite/satellitedb/projectaccounting.go]: we should add some tests that checks it.
look how it was tested in project_members
https://github.com/storj/storj/blob/master/satellite/console/projectmembers_test.go#L87

**Pull request merged**